skimage/morphology/rolling_ball.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+import numpy as np
+
+from skimage.util import invert
+
+
+def rolling_ball(input_img, radius=50, white_background=False):
+    """Perform background subtraction using the rolling ball method.
+
+    The rolling ball filter is a segmentation method that aims to separate the
```

Comment:
Hm, It does segment the image into "foreground" and "background", so that's why the phrasing is the way it currently is. Would foreground detection be more fitting? Which term did you have in mind?

Reply:
Segmentation usually refers to classifying *pixels* as foreground/background (or as different regions) rather than proportions of intensities.

"The rolling ball algorithm estimates image background intensity by..."

?

Reply:
true. I update the docstring.